src/query/service/tests/it/storages/fuse/bloom_index_meta_size.rs

```diff
@@ -63,64 +64,64 @@ use uuid::Uuid;
 //
 // please run the following two cases individually (in different process)
 
-#[tokio::test(flavor = "multi_thread")]
-#[ignore]
-async fn test_index_meta_cache_size_file_meta_data() -> databend_common_exception::Result<()> {
-    let thrift_file_meta = setup().await?;
+// #[tokio::test(flavor = "multi_thread")]
+// #[ignore]
```

Comment:
Is it possible to delete these codes if they are not useful anymore?

Reply:
This ut is a toolkit to decide cache which struct when reading bloom index. It can still be useful after bloom index is migrated to parquet-rs. So let's commet  for now.